src/protocols/rdp/rdp_settings.h

```diff
@@ -268,6 +268,20 @@ typedef struct guac_rdp_settings {
      */
     char** svc_names;
 
+    /**
+     * Whether outbound clipboard access should be blocked. If set, it will not
+     * be possible to copy data from the remote desktop to the client using the
+     * clipboard.
+     */
+    int disable_copy;
```

Comment:
As with the parsing, the other protocols use a `bool` type rather than an `int` - any particular reason for this?

Reply:
Yep - [see above](https://github.com/apache/guacamole-server/pull/214#discussion_r259652831).